mimic/test/test_core.py

```diff
@@ -55,3 +59,6 @@ def test_from_plugin_includes_all_plugins(self):
             len(plugin_apis),
             len(list(core.entries_for_tenant('any_tenant', {},
                                              'http://mimic'))))
+        self.assertEqual(
+            len(domain_apis),
+            len(list(core.domains)))
```

Comment:
I think the dummy plugin wasn't intended to be included in the default core. If we had any real domain plugins in this repo we could do this test. I have an idea for one.